cilium-ubuntu.json

```diff
@@ -134,7 +136,7 @@
       "box_tag": "cilium/ubuntu",
```

Comment:
This box-tag should be cilium/ubuntu-{{user `NAME_PREFIX`}}

Reply:
Yeah but I change it to `NAME_SUFFIX` for this case